queue/bytes_queue.go

```diff
@@ -14,9 +14,10 @@ const (
 )
 
 var (
-	errEmptyQueue       = &queueError{"Empty queue"}
+	errEmptyQueue       = &queueError{"Queue is empty."}
 	errInvalidIndex     = &queueError{"Index must be greater than zero. Invalid index."}
 	errIndexOutOfBounds = &queueError{"Index out of range"}
+	errFullQueue        = &queueError{"Queue is full. Maximum size limit reached."}
```

Comment:
:+1:

Reply:
```suggestion
	errFullQueue        = &queueError{"Full queue. Maximum size limit reached."}
```

backward compatibility